### PR TITLE
DC-1496: Update v2 url

### PR DESCRIPTION
**Pull request by matthewgroom**

### CODE REVIEW

File: conf/app1.routes [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# microservice specific routes
+
+GET        /inbox/v2                            uk.gov.hmrc.contactadvisors.controllers.SecureMessageController.inboxV2
+POST       /inbox/:utr             uk.gov.hmrc.contactadvisors.controllers.SecureMessageController.submit(utr: String)
```
Comment (felixgb) [inline conf/app1.routes]: Small thing but alignment here

File: conf/app1.routes [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# microservice specific routes
```
Comment (felixgb) [inline conf/app1.routes:1]: Small thing but could there be a more descriptive routes file name(s) now? something like `cafe.routes` and `secure-message.routes`?

**Review by felixgb — CHANGES_REQUESTED** [review]

File: conf/prod.routes [hunk 3]
```diff
@@ -1,6 +1,7 @@
 # Add all the application routes to the app.routes file
-->         /secure-message            app.Routes
-->         /                          health.Routes
-->     	   /template                  template.Routes
+->         /secure-message                        app.Routes
+->         /customer-advisors-frontend            app1.Routes
+->         /                                      health.Routes
+->     	   /template                              template.Routes
```
Comment (felixgb) [inline conf/prod.routes:5]: Looks like a hard tab here

**Review by felixgb — APPROVED** [review]

**Comment by vasanthavijaya** [comment]
Thanks Felix. Merging the request now.

**Pull request merged**